sr/LC_MESSAGES/django.po

```diff
@@ -372,7 +380,7 @@ msgstr ""
 #. Page path: /1-4/demo-tips/
 #: db-strings.txt:438 db-strings.txt.py:7098
 msgid "SijuLONPI9U"
-msgstr "Wur2jVusiDI"
+msgstr "SijuLONPI9U"
```

Comment:
Apologies that this is unclear, but these random looking strings are actually youtube ids and should not be translated to match. We should add a comment to that effect.